src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,21 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.exception.DukeException;
+
+public abstract class Command {
```

Comment:
I like how you have abstracted the Command class. It makes the code much easier to read and understand.